InformationScripting/src/dataformat/Tuple.cpp

```diff
@@ -24,40 +24,49 @@
 **
 ***********************************************************************************************************************/
 
-#include "InformationEdge.h"
+#include "Tuple.h"
 
 namespace InformationScripting {
 
-const QString InformationEdge::COUNT_PROPERTY_{"count"};
-const QString InformationEdge::NAME_PROPERTY_{"name"};
+Tuple::Tuple(std::initializer_list<NamedProperty> initialValue)
+	: values_{initialValue}
+{}
 
-InformationEdge::InformationEdge(InformationNode* from, InformationNode* to, const QString& name,
-											Orientation orientation)
-	: PropertyMap{{{COUNT_PROPERTY_, 1}, {NAME_PROPERTY_, name}}}, from_{from}, to_{to}, orientation_{orientation}
+Tuple::Tuple(const QList<NamedProperty>& initialValues)
+	: values_{initialValues}
 {}
 
-InformationEdge::InformationEdge(const InformationEdge& other)
-	: PropertyMap(other)
+NamedProperty Tuple::get(int index) const
+{
+	return values_.at(index);
+}
+
+QList<NamedProperty> Tuple::getAll(const QString& key)
```

Comment:
Should this return a list? A related question is: should we allow properties with the same name?